extract/db_download_files.py

```diff
@@ -0,0 +1,27 @@
+import dropbox
+from lib.get_dropbox import get_files, get_folders
+from lib.path import data_file_path
+
+
+with open(data_file_path("drop_box/token.txt", "r")) as f:
+    access_token = f.read()
```

Comment:
The best practice to deal with access tokens and secrets of any kind is to store them within `.env` file as environment variables. And then load them during script execution using package `dotenv`.

Reply:
Thanks.